src/ext/engine_hooks.h

```diff
@@ -77,6 +82,9 @@ inline void ddtrace_restore_error_handling(ddtrace_error_handling *eh TSRMLS_DC)
     PG(last_error_lineno) = eh->lineno;
     EG(error_reporting) = eh->error_reporting;
 }
+#else
+void ddtrace_restore_error_handling(ddtrace_error_handling *eh);
```

Comment:
Did this need to not be inline for some reason?

Reply:
Unfortunately this now has a call to `zend_string_release` which [is static](https://heap.space/xref/php-src/Zend/zend_string.h?r=573ad182#296) and causes a compiler warning about calling a static function from an inline function. This was my workaround to fix the warning, but if you have a better suggestion, I'm quite happy to accommodate! :)